rct229/utils/jsonpath_utils.py

```diff
@@ -11,30 +11,30 @@ def ensure_root(jpath):
     return jpath if jpath.startswith("$") else "$." + jpath
 
 
-def find_all(jpath, obj):
+def find_all(jpath, obj):  ####
```

Comment:
Please remove all the ```#```s. When we have a call, the reason why I added these was to let you know the functions we frequently used.